pytransifex/api.py

```diff
@@ -304,12 +323,12 @@ def pull(
 
     @ensure_login
     def push(
-        self, project_slug: str, resource_slugs: list[str], path_to_files: list[Path]
+        self, *, project_slug: str, resource_slugs: list[str], path_to_files: list[str]
```

Comment:
**Potential compatibility issue**: This function now disallows posittional arguments.